app/src/main/AndroidManifest.xml

```diff
@@ -15,6 +15,7 @@
         tools:targetApi="31">
         <activity
             android:name=".entry.EntryActivity"
+            android:windowSoftInputMode="stateAlwaysHidden"
```

Comment:
Why did you use this one?
Wouldn't this mean that you'll have to manually open the keyboard?